config/initializers/_constants.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+# Path to login via SSO authentication
+AUTH_PATH = '/auth/keycloak'
+
+# Callback path from SSO authentication
+AUTH_CALLBACK_PATH = "#{AUTH_PATH}/callback".freeze
+
+SESSION_DURATION_TIME = 3.hours
```

Comment:
Not sure if it is the best way (or at least the rails way) to centralize this kind of information